transaction/model.go

```diff
@@ -18,3 +18,31 @@ type APITx struct {
 	Nonce     uint64         `json:"nonce"`
 	Signature string         `json:"signature"`
 }
+
+// AtomicTx is a representation of a member of an atomic group
+type AtomicTx struct {
+	TxID      hezcommon.TxID `json:"id" binding:"required"`
+	Type      string         `json:"type"`
+	TokenID   uint32         `json:"tokenId"`
+	FromIdx   string         `json:"fromAccountIndex" binding:"required"`
+	ToIdx     string         `json:"toAccountIndex"`
+	ToEthAddr string         `json:"toHezEthereumAddress"`
+	ToBJJ     string         `json:"toBjj"`
+	Amount    string         `json:"amount" binding:"required"`
+	Fee       uint64         `json:"fee"`
+	Nonce     uint64         `json:"nonce"`
+	Signature string         `json:"signature"`
+	RqID      hezcommon.TxID `json:"requestId" binding:"required"`
+	RqOffSet  int            `json:"requestOffset" binding:"required"`
+}
+
+// AtomicGroup is a group of transactions that should run all or fail all
+type AtomicGroup struct {
+	AtomicGroupId hezcommon.TxID `json:"atomicGroupId" binding:"required"`
```

Comment:
This is not the right type, should be `common.AtomicGroupID`. But anyway, please use the struct from `api.AtomicGroup`, which includes the function `SetAtomicGroupID` as well as proper marshaling